test/verify/check-system-info

```diff
@@ -355,6 +355,12 @@ class TestSystemInfo(MachineCase):
         b.wait_visible('#motd-box')
         b.wait_text('#motd', "Hello again")
 
+        b.click("#motd-box-edit")
+        b.set_input_text("#motd-box-edit-modal textarea", "Hello cockpit team")
+        b.click("#motd-box-edit-modal button.pf-m-primary")
+        b.wait_not_present("motd-box-edit-modal")
+        b.wait_text('#motd', "Hello cockpit team")
```

Comment:
Can you please check here that `/etc/motd` also contains this new text and it hasn't just changed in cockpit?